tests/app/views/contexts/summary/test_question.py

```diff
@@ -141,6 +64,157 @@ def get_question_schema(answer_schema):
+                                                    "mandatory": True,
+                                                    "type": "TextField",
+                                                },
+                                                {
+                                                    "id": "address-line-2",
+                                                    "label": "Address Line 2",
+                                                    "mandatory": False,
+                                                    "type": "TextField",
+                                                },
+                                                {
+                                                    "id": "address-line-3",
+                                                    "label": "Address Line 3",
+                                                    "mandatory": False,
+                                                    "type": "TextField",
+                                                },
+                                                {
+                                                    "id": "town-city",
+                                                    "label": "Town/City",
+                                                    "mandatory": False,
+                                                    "type": "TextField",
+                                                },
+                                                {
+                                                    "id": "county",
+                                                    "label": "County",
+                                                    "mandatory": False,
+                                                    "type": "TextField",
+                                                },
+                                                {
+                                                    "id": "postcode",
+                                                    "label": "Postcode",
+                                                    "mandatory": False,
+                                                    "type": "TextField",
+                                                },
+                                                {
+                                                    "id": "country",
+                                                    "label": "Country",
+                                                    "mandatory": False,
+                                                    "type": "TextField",
+                                                },
+                                            ],
+                                        },
+                                    },
+                                ],
+                                "id": "address-group",
+                            }
+                        ],
+                    }
+                ]
+            }
+        )
+
+    def address_question(self, concatenation_type):
```

Comment:
Aren't these answers just a copy of address_questionnaire_schema?

Reply:
using the ones in `address_questionnaire_schema` now, no repetition. Thanks!